Gruntfile.js

```diff
@@ -146,6 +146,7 @@ module.exports = function(grunt) {
 
   grunt.registerTask('concat-core-sdk', ['jshint',  'concat:lawnchair', 'concat:crypto', 'browserify:dist']);
 
+  grunt.registerTask('build', ['concat-core-sdk', 'uglify:dist']);
```

Comment:
I added an explicit build task, because in our test client, I'm pointing directly at github, and hence I needed a command to build. (Running through all the tests wasn't always working when being built in the post-install script from another project.) This is not strictly needed for the factory function.